### ISSUE TITLE
[COGNITION]: EYB - CONSIDER hiding the HR divider from screen readers

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#9245]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️  Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The Your estimated benefits has an HR between the sections that is being picked up as an empty tab stop by screen readers. I'd like to hide this to make a small cognitive improvement. Screen shot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] The HR is ignored by screen readers. Users should be able to arrow without their virtual cursor stopping on the HR.

## Possible Fixes (optional)
```diff
- <hr />
+ <hr aria-hidden="true" role="presentation" />
```

## Screenshots or Trace Logs

![Screen Shot 2020-05-18 at 9 39 22 PM](https://user-images.githubusercontent.com/934879/82279275-c3869800-9951-11ea-966f-bf04e78a3379.png)

**Comment by 1Copenut** [comment]
@desireeturner This is an Estimate your benefits issue, my mistake in cataloging it as Section 103. I'll update the labels and issue title, but leave it in the current epic unless you want to move it.

**Comment by zurbergram** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/12830

**Comment by delli-gatti-michael** [comment]
This appears to be fixed on the above branch for regular profile pages, but VETTEC profiles have the same issue.

![Screen Shot 2020-06-02 at 4.10.32 PM (3).png](https://images.zenhubusercontent.com/5c07f000391a6f6357adc430/a7cde37a-b883-4d9c-9330-8742620968ed)

**Comment by delli-gatti-michael** [comment]
@1Copenut This fix is in Staging and ready for review

**Comment by 1Copenut** [comment]
@delli-gatti-michael Verified on a college detail and a VETTEC detail. HRs are hidden and no axe errors shown. Moving to close.
